hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -165,6 +168,9 @@ public class SingleTransactionRecordBuilderImpl
     // These are not persisted to the record file
     private final Map<AccountID, AccountID> deletedAccountBeneficiaries = new HashMap<>();
 
+    @Nullable
```

Comment:
A comment here to explain this field would be great.

Reply:
Ah yes, adding in current branch. 💯

Reply:
Oh wait that field is gone in my current branch. 🤷